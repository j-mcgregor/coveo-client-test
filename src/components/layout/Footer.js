import React from 'react';
import S from '../../static/styles';
import logo from '../../static/images/saq-logo.png';

const Footer = () => (
  <S.Footer className="container p-0">
    <div className="columns">
      <div className="column">
        <div className="hero hero-sm bg-dark">
          <div className="hero-body ">
            <div className="container grid-lg">
              <div className="columns">
                <div className="column d-flex image-container">
                  <img src={logo} alt="logo" className="logo" />
                </div>
                <div className="column col text-right">
                  <dl>
                    <dd>
                      <a
                        href="http://github.com/patchyj"
                        target="_blank"
                        rel="noopener noreferrer"
                      >
                        patchyj <i className="fab fa-github" />
                      </a>
                    </dd>
                    <dd>
                      <a
                        href="http://www.jackjwmcgregor.com"
                        target="_blank"
                        rel="noopener noreferrer"
                      >
                        jackjwmcgregor.com <i className="fas fa-globe" />
                      </a>
                    </dd>
                    <dd>
                      <a href="mailto:jackjwmcgregor@gmail.com">
                        jackjwmcgregor@gmail.com{' '}
                        <i className="fas fa-envelope" />
                      </a>
                    </dd>
                  </dl>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </S.Footer>
);

export default Footer;
