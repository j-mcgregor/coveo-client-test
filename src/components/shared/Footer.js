import React from 'react';

const Footer = () => (
  <div className="container p-0">
    <div className="columns">
      <div className="column">
        <div className="hero hero-sm bg-dark">
          <div className="hero-body">
            <div className="container grid-lg">
              <p className="text-center">This is a hero example</p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default Footer;
