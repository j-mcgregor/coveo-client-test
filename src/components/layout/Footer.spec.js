import { shallow } from 'enzyme';
import React from 'react';
import Footer from './Footer';

describe('Footer', () => {
  it('should render the Footer', () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});
