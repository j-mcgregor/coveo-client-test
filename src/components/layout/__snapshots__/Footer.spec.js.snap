// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render the Footer 1`] = `ShallowWrapper {}`;
